src/components/InputGroup.jsx

```diff
@@ -0,0 +1,34 @@
+import { Input } from "./Input";
+
+export function InputGroup({ handleChange }) {
+  return (
```

Comment:
Prilagodio sam samo prvi input kako bi treba izgledat da bi bio fleksibilna komponenta
```suggestion

const [initalInvestmant, setInitialInvestment] = useState(0);
return (

    <section id="user-input">
      <ul className="input-group">
        <li>
          <Input
            inputType="number"
            inputName="initial investment"
            handleInputChange={setInitialInvestment}
            inputValue={initialInvestment}
          ></Input>
          <Input
            inputType="number"
            inputName="expected return"
            handleInputChange={handleChange}
          ></Input>
        </li>
        <li>
          <Input
            inputType="number"
            inputName="annual investment"
            handleInputChange={handleChange}
          ></Input>
          <Input
            inputType="number"
            inputName="duration"
            handleInputChange={handleChange}
          ></Input>
        </li>
      </ul>
    </section>
```